test/jdk/sun/security/ssl/SSLSocketImpl/ReverseNameLookup.java

```diff
@@ -86,8 +86,11 @@
     void doServerSide() throws Exception {
         SSLServerSocketFactory sslssf =
             (SSLServerSocketFactory) SSLServerSocketFactory.getDefault();
+        InetSocketAddress socketAddress =
+                new InetSocketAddress(InetAddress.getLoopbackAddress(), serverPort);
```

Comment:
The client is using 127.0.0.1. InetAddress.getLoopbackAddress() might return ::1 (IPv6 loopback) on certain configurations. 
Two possibility:
  1. run this test with -Djava.net.preferIPv4Stack to force usage of IPv4, but trivially pass if IPv4 is not available on the machine (see IPSupport in the test library)
  2. use InetAddress.getLoopbackAddress().getHostAddress() on the client side instead of "127.0.0.1" 

If you choose 2. you may want to add an additional @run the test with -Djava.net.preferIPv6Address to verify that it works with IPv6 - as there are several ways to represent the IPv6 loopback as a string

Reply:
Applied the first suggestion since the it is using 127.0.0.1 for verification later in the test